backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -70,7 +71,7 @@ private void init() {
    */
   @PutMapping("/link/verify")
   public PxpPersonWrapper getPatientLinkVerify(@RequestBody PxpApiWrapper<Void> body)
-      throws InvalidPatientLinkException {
+      throws InvalidPatientLinkException, ExpiredPatientLinkException {
```

Comment:
You could also just remove the `throws` clause since these are both `RuntimeException`s. I'm not a big fan of using this syntax unless the compiler tells me I have no choice, but that's just a personal preference.

Reply:
Ah, yeah. Agreed & updated!